### PR TITLE
AEROGEAR 2956 - Update home screen

### PR DESCRIPTION
**Pull request by ciaranRoche**
## Motivation

JIRA: https://issues.jboss.org/browse/AEROGEAR-2956

## Progress

- [x] Replace Icon
- [x] Replace Text

## Screenshot
homescreen

### CODE REVIEW

**Comment by wtrocki** [comment]
Let's replace mobile `taskes` to tasks

File: src/pages/home/home.html [hunk 1]
```diff
@@ -10,12 +10,12 @@
...
   <ion-grid style="height: 100%">
-    <ion-row justify-content-center align-items-center style="height: 50%">
-      <img src="./assets/imgs/ag-logo.png" class="logo" height="90px" width="90px">
+    <ion-row justify-content-center align-items-center style="height: 55%">
+      <img src="./assets/imgs/aerogear_logo_600px.svg" class="logo" height="100px" width="220px">
     </ion-row>
     <ion-row justify-content-center align-items-center style="height: 1%">
       <ion-label color=stable text-wrap>
-        <div text-center>A Hybrid Cordova Application for iOS and Android that showcases functionality</div>
+        <div text-center>This app demonstrates how to use the AeroGear Mobile services, backed by OpenShift to perform a wide range of common mobile tasks such as Identity Management, Device Security, Metrics Capture, Push Notifications and more. For further information, visit <a href="https://www.aerogear.org/">aerogear.org</a></div>
```
Comment (wtrocki) [inline src/pages/home/home.html]: [Optional] Not sure what is requirement here but do we want this website to open in the phone browser or in inappbrowser (inside the app)?

Comment (wei-lee) [reply src/pages/home/home.html]: phone browser is ok

Comment (ciaranRoche) [reply src/pages/home/home.html]: It is not specified which one to use, so I left it open to allow the user choose the browser, maybe something @wei-lee or @laurafitzgerald can confirm for us

Comment (wtrocki) [reply src/pages/home/home.html]: @ciaranRoche That is 100% ok and expected behavior. Got that on the Android. Testing that now in IOS and going to approve it.

**Comment by wtrocki** [comment]
@ciaranRoche It worked on IOS without last commit. Is there any problem?

**Comment by ciaranRoche** [comment]
@wtrocki No problem, just updated some wording to match the wording that got merged to the IOS-showcase-app

**Pull request merged**